.spectral.yml

```diff
@@ -15,6 +50,56 @@ rules:
       functionOptions:
         notMatch: "^(options|head|trace)$"
 
+  # Data Format - JSON support requirement
+  entur-request-media-type:
+    severity: error
+    message: "Request MUST support `application/json` media type."
+    given: $.paths.*.*.requestBody
+    then:
+      field: content.application/json
+      function: defined
+
+  # Documentation with examples
+  entur-example-parameter:
+    message: "Parameters SHOULD have example values."
+    severity: warn
+    recommended: false
+    given: $.paths.*.*.parameters.*
+    then:
+      field: example
+      function: defined
+
+  entur-example-schema-property:
+    message: "Properties in schema SHOULD have example values."
+    severity: warn
+    recommended: false
+    given: $.components.schemas.*.properties.*
+    then:
+      field: example
+      function: defined
+
+  entur-example-schema:
+    message: "Schemas SHOULD have an example."
+    severity: warn
+    recommended: false
+    given: $.components.schemas.*
+    then:
+      field: example
+      function: defined
+
+  # openapi spec version 3
```

Comment:
Checks in rule entur-openapi-version-3 is covered by the rule unrecognized-format, so I don't think it is needed.

Reply:
updated with next commit

Reply:
Ahaa, right. Nice